src/main/java/net/minecraftforge/fml/client/config/HoverChecker.java

```diff
@@ -19,6 +19,7 @@
 
 package net.minecraftforge.fml.client.config;
 
+import net.minecraft.client.gui.widget.Widget;
```

Comment:
Can the other import for button be removed now? Glancing at the class it seems like it could be but didn't open up a workspace to verify